src/main/resources/GitHub_True_Dark.xml

```diff
@@ -0,0 +1,576 @@
+<scheme name="GitHub True Dark" version="142" parent_scheme="Darcula">
+    <metaInfo>
+        <property name="created">2021-08-05T08:57:01</property>
```

Comment:
update with date when you submitted PR

Reply:
Sorry that is unclear to me right now, is it enough to just insert the pull request tag here or does it need to be specific with timestamp?

Reply:
Just put current date in same format thats it.